table/queries/derived_satellite_scans.sql

```diff
@@ -1,52 +1,52 @@
-    WHEN rcode = 6 THEN "❗️dns/rcode:YXDomain"
-    WHEN rcode = 7 THEN "❗️dns/rcode:YXRRSet"
-    WHEN rcode = 8 THEN "❗️dns/rcode:NXRRSet"
-    WHEN rcode = 9 THEN "❗️dns/rcode:NotAuth"
-    WHEN rcode = 10 THEN "❗️dns/rcode:NotZone"
-    WHEN rcode = 11 THEN "❗️dns/rcode:DSOTYPENI"
-    WHEN rcode = 12 THEN "❗️dns/rcode:Unassigned"
-    WHEN rcode = 13 THEN "❗️dns/rcode:Unassigned"
-    WHEN rcode = 14 THEN "❗️dns/rcode:Unassigned"
-    WHEN rcode = 15 THEN "❗️dns/rcode:Unassigned"
-    WHEN rcode = 16 THEN "❗️dns/rcode:BadVers"
-    WHEN rcode = 17 THEN "❗️dns/rcode:BadSig"
-    WHEN rcode = 18 THEN "❗️dns/rcode:BadKey"
-    WHEN rcode = 19 THEN "❗️dns/rcode:BadTime"
-    WHEN rcode = 20 THEN "❗️dns/rcode:BadMode"
-    WHEN rcode = 21 THEN "❗️dns/rcode:BadAlg"
-    WHEN rcode = 22 THEN "❗️dns/rcode:BadTrunc"
-    WHEN rcode = 23 THEN "❗️dns/rcode:BadCookie"
-    ELSE CONCAT("❗️dns/unknown_rcode:", rcode)
+    WHEN rcode = 4 THEN "⚠️dns/rcode:NotImp"
+    WHEN rcode = 5 THEN "⚠️dns/rcode:Refused"
+    WHEN rcode = 6 THEN "⚠️dns/rcode:YXDomain"
+    WHEN rcode = 7 THEN "⚠️dns/rcode:YXRRSet"
+    WHEN rcode = 8 THEN "⚠️dns/rcode:NXRRSet"
+    WHEN rcode = 9 THEN "⚠️dns/rcode:NotAuth"
+    WHEN rcode = 10 THEN "⚠️dns/rcode:NotZone"
+    WHEN rcode = 11 THEN "⚠️dns/rcode:DSOTYPENI"
+    WHEN rcode = 12 THEN "⚠️dns/rcode:Unassigned"
+    WHEN rcode = 13 THEN "⚠️dns/rcode:Unassigned"
+    WHEN rcode = 14 THEN "⚠️dns/rcode:Unassigned"
+    WHEN rcode = 15 THEN "⚠️dns/rcode:Unassigned"
+    WHEN rcode = 16 THEN "⚠️dns/rcode:BadVers"
+    WHEN rcode = 17 THEN "⚠️dns/rcode:BadSig"
+    WHEN rcode = 18 THEN "⚠️dns/rcode:BadKey"
+    WHEN rcode = 19 THEN "⚠️dns/rcode:BadTime"
+    WHEN rcode = 20 THEN "⚠️dns/rcode:BadMode"
+    WHEN rcode = 21 THEN "⚠️dns/rcode:BadAlg"
+    WHEN rcode = 22 THEN "⚠️dns/rcode:BadTrunc"
+    WHEN rcode = 23 THEN "⚠️dns/rcode:BadCookie"
+    ELSE CONCAT("⚠️dns/unknown_rcode:", rcode)
   END
 );
 
 CREATE TEMP FUNCTION ClassifySatelliteError(error STRING) AS (
   CASE
     # Satellite v1
-    WHEN REGEXP_CONTAINS(error, '"Err": {}') THEN "❗️read/udp.timeout"
-    WHEN REGEXP_CONTAINS(error, '"Err": 90') THEN "❗️read/dns.msgsize"
-    WHEN REGEXP_CONTAINS(error, '"Err": 111') THEN "❗️read/udp.refused"
-    WHEN REGEXP_CONTAINS(error, '"Err": 113') THEN "❗️read/ip.host_no_route"
-    WHEN REGEXP_CONTAINS(error, '"Err": 24') THEN "❗️setup/system_failure" # Too many open files
-    WHEN error = "{}" THEN "❗️dns/unknown" # TODO figure out origin
-    WHEN error = "no_answer" THEN "❗️dns/answer:no_answer"
+    WHEN REGEXP_CONTAINS(error, '"Err": {}') THEN "⚠️read/udp.timeout"
+    WHEN REGEXP_CONTAINS(error, '"Err": 90') THEN "⚠️read/dns.msgsize"
+    WHEN REGEXP_CONTAINS(error, '"Err": 111') THEN "⚠️read/udp.refused"
+    WHEN REGEXP_CONTAINS(error, '"Err": 113') THEN "⚠️read/ip.host_no_route"
+    WHEN REGEXP_CONTAINS(error, '"Err": 24') THEN "⚠️setup/system_failure" # Too many open files
```

Comment:
For these two, wouldn't it be safe to give a different emoji that indicates it could a measurement problems? i.e. in these cases, there is no way one can figure out whether it is censorship according to our measurements. For the others, we are capturing something, but just not sure about what we are capturing. I'm thinking of 4 divisions kind of like OONI's mechanism:
1. Okay
2. Our error
3. Possible censorship but cant be sure
4. Censorship

These two could be assigned code 2.

Reply:
If we think something is our error we should just filter it out and not show it in the dashboard.

We actually are filtering out `setup`
https://github.com/censoredplanet/censoredplanet-analysis/pull/187/files#diff-20fce2db9fd5531520462ed659edd5c769af3568b7ff5b03617c44be1434f3d0L156

I've removed the emoji so that will match correctly.

We could remove other things like `ip.host_no_route` as well, but I'd rather do that in a separate change supported by what we're seeing in the data.